### PR TITLE
Updated Master Driver docs structure in RTD

### PR DESCRIPTION
**Pull request by jklarson**
# Description

Updated the structure of the Master driver docs in agent and driver sections per feedback.  Includes fixes for a few more minor comments.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Tested using local Sphinx build

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: docs/source/agent-framework/core-service-agents/master-driver/master-driver-agent.rst [hunk 1]
```diff
@@ -14,16 +14,9 @@ Master Driver Agent.  For more information on the Master Driver Agent's operatio
...
-
-* Master Driver Agent configuration file - lists all driver configuration files to load
-* Driver configuration file - contains the general driver configuration and device settings
-* Device Register configuration file - contains the settings for each individual data point on the device
-
-For each device, you must create a driver configuration file, device register configuration file, and an entry in the
-Master Driver Agent configuration file.
-
-Once configured, the Master Driver Agent is configured and deployed in a manner similar to any other agent:
+The Master Driver like all other agents, requires a configuration file (described in brief below).  Once the user
```
Comment (craig8) [inline docs/source/agent-framework/core-service-agents/master-driver/master-driver-agent.rst:17]: remove like all other agents...

File: docs/source/agent-framework/core-service-agents/master-driver/master-driver-agent.rst [hunk 2]
```diff
@@ -46,314 +39,32 @@ Required libraries:
...
 
 Master Driver Agent Configuration
 ---------------------------------
 
 The Master Driver Agent configuration consists of general settings for all devices. The default values of the Master
-Driver should be sufficient for most users.  The user may optionally change the interval between device scrapes with the
-driver_scrape_interval.
-
-The following example sets the driver_scrape_interval to 0.05 seconds or 20 devices per second:
+Driver should be sufficient for most users.
```
Comment (craig8) [inline docs/source/agent-framework/core-service-agents/master-driver/master-driver-agent.rst:51]: get rid of should be ...

just add as follows or some other wording that says look at the example.

File: docs/source/agent-framework/core-service-agents/master-driver/master-driver-agent.rst (hunk 1 above)
Comment (craig8) [reply docs/source/agent-framework/core-service-agents/master-driver/master-driver-agent.rst:17]: Add configuration file to set global settings for all drivers.

File: docs/source/agent-framework/core-service-agents/master-driver/master-driver-agent.rst [hunk 3]
```diff
@@ -14,16 +14,9 @@ Master Driver Agent.  For more information on the Master Driver Agent's operatio
...
-* Driver configuration file - contains the general driver configuration and device settings
-* Device Register configuration file - contains the settings for each individual data point on the device
-
-For each device, you must create a driver configuration file, device register configuration file, and an entry in the
-Master Driver Agent configuration file.
-
-Once configured, the Master Driver Agent is configured and deployed in a manner similar to any other agent:
+The Master Driver like all other agents, requires a configuration file (described in brief below).  Once the user
+has copied the example or created their own config, the Master Driver Agent is deployed in a manner similar to any other
+agent:
```
Comment (craig8) [inline docs/source/agent-framework/core-service-agents/master-driver/master-driver-agent.rst:19]: Remove similar to any other agent.

Just is deployed using the following:

**Review by craig8 — CHANGES_REQUESTED** [review]

**Review by shwethanidd — APPROVED** [review]

**Pull request merged**